Application/EdFi.Ods.Features.OwnershipBasedAuthorization/NHibernate/OwnershipBasedAuthorizationNHibernateConfigurationActivity.cs

```diff
@@ -58,7 +58,8 @@ public void Execute(object sender, BindMappingEventArgs e)
         private static bool IsEdFiQueryMappingEvent(BindMappingEventArgs e)
         {
             return (e.Mapping.@namespace.Equals(Namespaces.Entities.NHibernate.QueryModels.BaseNamespace) ||
-                    e.Mapping.@namespace.Equals(Namespaces.Entities.NHibernate.QueryModels.Views))
+                    e.Mapping.@namespace.Equals(Namespaces.Entities.NHibernate.QueryModels.Views) ||
+                    e.Mapping.@namespace.Equals(Namespaces.Entities.NHibernate.QueryModels.Tables))
                    && e.Mapping.assembly.Equals(Namespaces.Standard.BaseNamespace);
```

Comment:
This change (and the matching one on ChangeQueries) is a good indicator that this isn't really implementation specific code, and should be exposed in the core libraries.  Convert this boolean method into an extension method on BindMappingEventArgs, and then change there and ChangeQueries to consume it, so we can manage what are "Ed-Fi" query mapping events in one place.